recipes-ni/niacctbase/niacctbase/sudoers

```diff
@@ -0,0 +1,6 @@
+
+# Allow the admin user (UID 0) to invoke sudo without a password
+admin ALL=(ALL) NOPASSWD: ALL
+
+# Allow lvuser to modify the opkg state without a password
+lvuser ALL=(#0:#0) NOPASSWD: /usr/bin/opkg info *
```

Comment:
NOTE: I'm not actually sure this rule is needed, now that opkg will allow unpriviledged users to perform read-only operations. But it isn't worth the effort at the moment for me to track down this requirement.